hw3/js/hw3-7.js

```diff
@@ -0,0 +1,14 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const firstNumber = prompt('Enter the first number', '');
```

Comment:
You can combine methods to avoid redundant variables
```suggestion
    const firstNumber = Number(prompt('Enter the first number', ''));
```